testbed/tests/scenarios.go

```diff
@@ -51,12 +51,12 @@ func createConfigYaml(
 	processorsList := ""
 	if len(processors) > 0 {
 		first := true
-		for name, cfg := range processors {
-			processorsSections += cfg + "\n"
+		for i := range processors {
+			processorsSections += processors[i][1] + "\n"
```

Comment:
this calls for the use of a struct rather than a tuple in a string of string slice, imo.

Reply:
Something like this?
```go
type ProcessorConfig struct {
	name       string
	configBody string
}

processors := []ProcessorConfig{
	ProcessorConfig{
		"batch",
		`
  batch:
`,
	},
}
```

or like this?
```go
type ProcessorConfigs struct {
	names       []string
	configBodies []string
}
```

Reply:
The former.

Reply:
@atoulme
Updated. Plz review now.